src/game/mapitems.h

```diff
@@ -54,7 +54,16 @@ enum
 	TILEFLAG_ROTATE=8,
 
 	LAYERFLAG_DETAIL=1,
+	LAYERFLAG_OPERATIONAL=2,
+	LAYERFLAG_NO_IMAGE=4,
```

Comment:
Not sure I understand the purpose of these flags, at least in the current state. Right now they seem redundant, as you can derive them from the other `TILESLAYERFLAG`s and other fields. For example, if `m_Image < 0` then there is no image.

Reply:
@Robyt3 this are future placeholder for (the yet not implemented) custom layers. Custom layers should load their own entities and therefore have an Image. I think we can derive this information from the m_Image field 👍

Reply:
@Robyt3 I use the new flag LAYERFLAG_CUSTOM_GAMELAYER now (instead of NO_IMAGE) in order to detect if we have a custom layer and removed the old flag in the process.